### ISSUE TITLE
该方案产生的新问题

### ISSUE THREAD

**Issue by thirtyyuan** [hellosad/FixV26AppBarLayoutBounce#1]
感谢你的解决方案，但是又遇到下列问题：
1. 用以前会产生抖动的操作，现在 AppbarLayout 会在强行结束动画的时候直接回到顶部；
2. 在 RecyclerView 往上滑动的时候，对 AppBarLayout 下拉，会产生短暂的，类似之前的抖动效果。
目前我也正在尝试解决中。

**Comment by KaelInvoker** [comment]
有解决的办法么？我用了这个方案也不行

**Comment by thirtyyuan** [comment]
Nope, I didn't try to solve it again, but find some App just use NestedScrollView directly, rather than AppbarLayout.
